astropy/cosmology/flrw.py

```diff
@@ -415,7 +415,7 @@ def Om(self, z):
         """
 
         if isiterable(z):
```

Comment:
Looking at this again, I wonder about the use of `isiterable` here - after all, `u.Quantity(...)` can deal with a scalar as well...

Reply:
I have a card in the Cosmology project to look at this more carefully. The docstring and commit history lead me to believe its a performance issue.

Reply:
@nstarman I've done a rough benchmark of several of the methods with and without `if isiterable(z):` using the `timeit` library and it is notably faster with, so as you say it likely is a performance thing.

Reply:
OK, makes sense then to leave in place, at least for now! It is somewhat indirectly related to this PR anyway.

Reply:
Agreed.